server/src/main/resources/users.json

```diff
@@ -17,8 +17,8 @@
     "_id": "588935f51c55b55c75a84848",
     "name": "Roseann Roberson",
     "age": 23,
-    "company": "GINKLE",
-    "email": "roseannroberson@ginkle.com"
+    "company": "OHMNET",
+    "email": "roseannroberson@ohmnet.com"
```

Comment:
this change will effect the hash used for the avatar in #368 so we need to watch for this when we bring them together.